### ISSUE TITLE
Saving form template returns error

### ISSUE THREAD

**Issue by TomDeSmet** [verbb/formie#920]
**Description**
I'm trying to save a form template from the settings. When *use custom template* is on, a folder path is added, but the folder path does not exist yet this error is shown:

![image](https://user-images.githubusercontent.com/16959207/171209247-bc9010e1-2d0f-4fb3-a882-ceb6dfaf0a9b.png)

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Both
- Client-side Validation: Both
- Custom Form Templates: No

**Additional info**
- Plugin version: 2.0.0-beta.10
- Craft version: 4.0.3 Pro
- Multi-site: No

**Comment by engram-design** [comment]
Fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.10",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.11](https://github.com/verbb/formie/releases/tag/2.0.0-beta.11)
